lib/requirements/preview/edition_checker.rb

```diff
@@ -0,0 +1,23 @@
+class Requirements::Preview::EditionChecker < Requirements::Checker
+  attr_reader :edition
+
+  CHECKERS = [
+    Requirements::Preview::ContentChecker,
+    Requirements::Preview::TagsChecker,
+    Requirements::Preview::ImagesChecker,
+  ].freeze
```

Comment:
I wonder if we might end up seeing some autoloading problems with this approach at some point - feels like the sort of thing that may lead to a cyclic dependency.

No problem with it going in now but may need to consider using strings or an instance method if we hit it.

Reply:
Sure, I see what you mean. I do prefer using the constant name, as then we get a clearer error if it's been mis-typed. Like you say, we can always fix it if need be.

Reply:
Yup, that's a good point about catching the undefined constant. Lets just cross our fingers no dependencies become cyclic 🤞